src/lib/routes/edge-api/index.ts

```diff
@@ -78,23 +85,24 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary:
+                        'Accepts batched metrics from Edge, for insertion into our metrics storage',
                     operationId: 'bulkMetrics',
                     requestBody: createRequestSchema('bulkMetricsSchema'),
                     responses: {
                         202: emptyResponse,
+                        ...getStandardResponses(400),
```

Comment:
Don't we have any permission validation that could yield a 401? I noticed [this comment](https://github.com/Unleash/unleash/blob/19d318d684a63c5690c54995f5103625499733dc/src/lib/routes/edge-api/index.ts#L77) so maybe is something we missed? I think the idea was to do a manual validation, but it seems we didn't do it

Reply:
We don't do auth here, so no 401,403. But Thomas is right 415 is a possibility, we don't accept any other content types than application/json. Technically I guess we could yield 413 as well if you try to post more than 100kB of metrics at the time